src/index.ts

```diff
@@ -104,21 +105,29 @@ export class WebRTCPlayer extends EventEmitter {
     }
   }
 
-  private async onConnectionStats() {
+  private async onConnectionStats() { 
+    
     if (this.peer && this.statsTypeFilter) {
+      let bytesReceivedBlock: number = 0;
       let stats = await this.peer.getStats(null);
       stats.forEach((report) => {
         if (report.type.match(this.statsTypeFilter)) {
           this.emit(`stats:${report.type}`, report);
         }
 
         if (report.type.match('inbound-rtp')) {
-          if (report.bytesReceived === '0'){
             this.emit(`stats:${report.type}`, report);
-          }
+            bytesReceivedBlock += report.bytesReceived;
         }
 
       });
+
+      if (bytesReceivedBlock <= this.bytesReceived) {
+        this.emit('media reception ended');
```

Comment:
Is there a naming convention for events? Seems like something less like a description and more like a single word would be good. 'no-media' for example.